cdap-docs/cdap-apps/source/hydrator/creating.rst

```diff
@@ -69,6 +86,12 @@ The application launches a MapReduce program that runs every minute, reads data
 stream *myStream* and writes to a Table *myTable*. A Table Sink needs a row key field to
 be specified and can use the timestamp of a Stream event for that.
 
+Pipeline config consists of stages - source, zero (or) more transforms and one (or) more sinks.
+Each of these stages are identified by a unique name and plugin.
+Plugin object consists of plugin-name, properties map and optionally can specify the artifact.
```

Comment:
since this PR is about updating the config, i didn't describe those here. can include them in detail in DAG transform PR. https://github.com/caskdata/cdap/pull/4873